migrations/20190321232837_create_users_table.js

```diff
@@ -0,0 +1,29 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.withSchema('public').createTable('users', function(table) {
```

Comment:
```suggestion
  return knex.schema.createTable('users', function(table) {
```
Here and in other places, you file the new tables under a schema. If it isn't required (I don't this it is for PostgreSQL) Our project doesn't look like it'll be complex enough to need to sort our tables into schemas. This will make it slightly harder (and easy to forget how) to query a table when it needs to be referred to as `public.memes`, or we need to use withSchema every query. https://knexjs.org/#Builder-withSchema

Reply:
Updated.